### PR TITLE
WT-6281 salvage/verify should not assert or panic in the case of corrupted page cells

### PR DESCRIPTION
**Pull request by keithbostic**

### CODE REVIEW

File: src/include/cell.i [hunk 1]
```diff
@@ -10,18 +10,20 @@
...
 #ifdef HAVE_DIAGNOSTIC
     WT_DECL_RET;
 
     if ((ret = __wt_time_value_validate(session, tw, NULL, false)) != 0)
-        WT_IGNORE_RET(__wt_panic(session, ret, "value timestamp window failed validation"));
+        return (expected_error ? WT_ERROR : __wt_panic(session, ret,
+                                              "value timestamp window failed validation"));
 #else
     WT_UNUSED(session);
     WT_UNUSED(tw);
```
Comment (hbokhari) [inline src/include/cell.i:24]: Add WT_UNUSED() for `expected_error`.

File: src/include/cell.i [hunk 2]
```diff
@@ -98,18 +100,20 @@ __cell_pack_value_validity(
...
 #ifdef HAVE_DIAGNOSTIC
     WT_DECL_RET;
 
     if ((ret = __wt_time_aggregate_validate(session, ta, NULL, false)) != 0)
-        WT_IGNORE_RET(__wt_panic(session, ret, "illegal address timestamp window"));
+        return (expected_error ? WT_ERROR : __wt_panic(session, ret,
+                                              "address timestamp window failed validation"));
 #else
     WT_UNUSED(session);
     WT_UNUSED(ta);
```
Comment (hbokhari) [inline src/include/cell.i:114]: Add WT_UNUSED() for `expected_error`.

File: src/include/cell.i (hunk 1 above)
Comment (keithbostic) [reply src/include/cell.i:24]: Fixed in d9dff196f.

File: src/include/cell.i (hunk 2 above)
Comment (keithbostic) [reply src/include/cell.i:115]: Fixed in d9dff196f.

**Pull request merged**